client/src/components/screens/Patient/components/FamilyTab/components/AddParentModal.tsx

```diff
@@ -0,0 +1,135 @@
+import {
+  AutoComplete, Form, Radio,
+} from 'antd';
+import { useForm } from 'antd/lib/form/Form';
+import Modal from 'antd/lib/modal/Modal';
+import React, { useState } from 'react';
+import intl from 'react-intl-universal';
+import { useDispatch } from 'react-redux';
+import { addParentToFamily } from '../../../../../../actions/patient';
+import api from '../../../../../../helpers/api';
+import { FamilyMemberType, ParsedPatientData } from '../../../../../../helpers/providers/types';
+
+interface Props {
+  parentType: FamilyMemberType | null
+  onClose: () => void
+}
+
+interface SearchResult {
+  id: string
+  lastName: string
+  firstName: string
+}
+
+const AddParentModal: React.FC<Props> = ({
+  parentType,
+  onClose,
+}) => {
+  const dispatch = useDispatch();
+  const [form] = useForm();
+  const [searchResult, setSearchResult] = useState<SearchResult[] | null>(null);
+  const [selectedPatient, setSelectedPatient] = useState<ParsedPatientData | null>(null);
+
+  async function search(searchTerm: string) {
+    const response: any = await api.getPatientsByAutoComplete('partial', searchTerm, 1, 5);
+    if (response.payload?.data) {
+      setSearchResult(response.payload.data.data.hits
+        .map((hit: any) => ({ id: hit._id, lastName: hit._source.lastName, firstName: hit._source.firstName })));
+    } else {
+      setSearchResult([]);
+    }
+  }
+
+  async function onSelectPatient(value: string, option: any) {
+    const response: any = await api.getPatientById(option.key);
+    setSelectedPatient(response.payload.data.data);
```

Comment:
The data you're receiving here is of type `PatientData` from [here](https://github.com/Ferlab-Ste-Justine/clin-frontend/blob/master/client/src/helpers/search/types.ts#L15), so `selectedPatient` should be of type `PatientData` not `ParsedPatientData`